lib/wick-get-iface-ip

```diff
@@ -33,7 +33,17 @@ wickGetIfaceIp() {
         allInterfaces=true
     fi
 
-    if wickCommandExists ifconfig; then
+    # Works for systemd distros
+    if wickCommandExists networkctl; then
+        cmd=(networkctl status)
+
+	if [[ -n "$interface" ]]; then
+	    cmd[${#cmd[@]}]="$interface"
+	fi
+
+       # FIXME: Unfortunately, IPv4 only
+       list=( $("${cmd[@]}" 2>/dev/null | grep -oP "\b(?:[0-9]{1,3}\.){3}[0-9]{1,3}\b") )
```

Comment:
Son of a gun!  Good catch.  [Seems to have recently been broken](http://stackoverflow.com/questions/16658333/grep-p-no-longer-works-how-can-i-rewrite-my-searches) by a change from GNU grep to another grep.  I'll change the regex.